### PR TITLE
Prepare update to Django 4.2

### PR DESCRIPTION
**Pull request by Viicos**
Fixes maykinmedia/objects-api#385

### CODE REVIEW

**Comment by Viicos** [comment]
Waiting on a new `commonground-api-common` release.

File: src/objecttypes/conf/dev.py [hunk 1]
```diff
@@ -107,8 +107,9 @@
     r"django\.db\.models\.fields",
 )
 
-if "test" in sys.argv:
-    TWO_FACTOR_FORCE_OTP_ADMIN = False
+# None of the authentication backends require two-factor authentication.
+if config("DISABLE_2FA", default=True):  # pragma: no cover
```
Comment (annashamray) [inline src/objecttypes/conf/dev.py:111]: Could you please add `DISABLE_2FA` to base config in the same way as in Objects API? I think they should be consistent

**Review by annashamray — CHANGES_REQUESTED** [review]

File: src/objecttypes/conf/dev.py (hunk 1 above)
Comment (Viicos) [reply src/objecttypes/conf/dev.py:111]: done

**Review by annashamray — CHANGES_REQUESTED** [review]
Same error when fetching schema as it was in Objects API appear:
```
AttributeError: module 'rest_framework.serializers' has no attribute 'NullBooleanField'
```

**Pull request merged**